### DISCUSSION TITLE
nolint and EmbedLiveSample

### DISCUSSION THREAD

**Discussion by Josh-Cena** [mdn/mdn-community#266] [Browser compatibility data] [answered]
Page fails to build after mdn/content#21215:

<img width="498" alt="image" src="https://user-images.githubusercontent.com/55398995/195693992-e15ef4e0-27b5-4005-ab51-8e4f453a04cf.png">

Should we make `EmbedLiveSample` recognize `nolint` code blocks?

**Comment by schalkneethling** [comment]
Tagging @mdn/core-dev

**Comment by wbamberg** [comment]
> Should we make EmbedLiveSample recognize nolint code blocks?

Yes.

**Comment by caugner** [comment, marked as answer]
We are now tracking this in https://github.com/mdn/yari/issues/7379.

**Comment by pepelsbey** [comment]
It seems like the [bug has been filed](https://github.com/mdn/yari/issues/7379), and this discussion is no longer relevant. Closing as resolved.
